src/utils/wrappers.ts

```diff
@@ -386,21 +424,27 @@ export function handleWrappers(ast: Node) {
         wrapper.callee.body.body[3].expression.right.type === 'ObjectExpression' &&
         (wrapper.callee.body.body[4].type === 'VariableDeclaration' &&
           wrapper.callee.body.body[4].declarations.length === 1 &&
+          wrapper.callee.body.body[4].declarations[0].init &&
           wrapper.callee.body.body[4].declarations[0].init.type === 'CallExpression' &&
           wrapper.callee.body.body[4].declarations[0].init.callee.type === 'Identifier' &&
           wrapper.callee.body.body[4].declarations[0].init.callee.name === 'require' ||
           wrapper.callee.body.body[5].type === 'VariableDeclaration' &&
           wrapper.callee.body.body[5].declarations.length === 1 &&
+          wrapper.callee.body.body[5].declarations[0].init &&
           wrapper.callee.body.body[5].declarations[0].init.type === 'CallExpression' &&
           wrapper.callee.body.body[5].declarations[0].init.callee.type === 'Identifier' &&
           wrapper.callee.body.body[5].declarations[0].init.callee.name === 'require')) {
       const externalMap = new Map<number, any>();
-      const moduleObj = wrapper.callee.params.length ? wrapper.arguments[0] : wrapper.callee.body.body[3].expression.right;
-      let modules: [number, any][];
+      const statement = wrapper.callee.body.body[3];
+      if (statement.type !== 'ExpressionStatement' || statement.expression.type !== 'AssignmentExpression' || statement.expression.right.type !== 'ObjectExpression') {
+        throw new Error('Expected ExpressionStatement');
+      }
+      const moduleObj = wrapper.callee.params.length ? wrapper.arguments[0] : statement.expression.right;
```

Comment:
I'm wondering if the bug is here because `wrapper.arguments[0]` can basically be anything but `statement.expression.right` can only be `ObjectExpression`.

Do we need to narrow down `wrapper.arguments[0]` to a specific type?